test/claim-service.test.js

```diff
@@ -0,0 +1,81 @@
+
+describe('Test claim service', () => {
+  let claimService
+  let mockClaimRepository
+  let mockMinetypeRepository
+  let mockMessageService
+
+  beforeAll(async () => {
+    jest.mock('../server/repository/claim-repository')
+    jest.mock('../server/repository/minetype-repository')
+    jest.mock('../server/services/message-service')
+  })
+
+  beforeEach(async () => {
+    jest.resetModules()
+    mockClaimRepository = require('../server/repository/claim-repository')
+    mockMinetypeRepository = require('../server/repository/minetype-repository')
+    mockMessageService = require('../server/services/message-service')

```

Comment:
The mocks can all be required once in `beforeAll` (immediately after the `jest.mock()` calls).

Reply:
No, they can't. If you do that the test fails. Likely because of the reset of the module between each test. So you have to do a require in each test.

Reply:
Sorry, I lost track of that comment after I realised I was running the tests against old code when I tried out that change.